src/construction-site.ts

```diff
@@ -24,7 +32,7 @@ interface ConstructionSite<T extends BuildableStructureConstant = BuildableStruc
      */
     progressTotal: number;
     /**
-     * One of the `STRUCTURE_*` constants.
+     * One of the {@link StructureConstant} constants.
      */
```

Comment:
```suggestion
     * @see {@link StructureConstant} .
```
How about keeping the `STRUCTURE_*` comment

Reply:
I thought the links being clickable would be an improvement. That's the rationale for me changing those `THING_*` plain text into something VSCode shows as a link to the full list.

Reply:
let's use ```{@link StructureConstant `STRUCTURE_*`}```